listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -659,7 +659,7 @@ export default class RecentListens extends React.Component<
       <ul className="pager" style={{ display: "flex" }}>
         <li
           className={`previous ${
-            currRecPage && currRecPage <= 1 ? "hidden" : ""
+            currRecPage && currRecPage <= 1 ? "disabled" : ""
```

Comment:
rebase over master please. I think by mistake you made a branch from the other PR branch